### ISSUE TITLE
new bug on "Plugin load failed: %s hexo-theme-landscape"

### ISSUE THREAD

**Issue by 755452800** [hexojs/hexo#4594]
## Check List

Please check followings before submitting a new issue.

- [x] I have already read [Docs page](https://hexo.io/docs/) & [Troubleshooting page](https://hexo.io/docs/troubleshooting)
- [x] I have already searched existing issues and they are not help  to me
- [x] I examined error or warning messages and it's difficult to solve
- [x] Using [the latest](https://github.com/hexojs/hexo/releases) version of Hexo (run `hexo version` to check)
- [x] Node.js is higher than [minimum required version](https://hexo.io/docs/#Minimum-required-Node-js-version)

## Expected behavior
run smoothly without error
## Actual behavior
run command hexo s, hexo g, hexo clean will outcome an same error as follow. **Although an error occurred, it worked properly.**

ERROR {
  err: [Error: EISDIR: illegal operation on a directory, read] {
    errno: -4068,
    code: 'EISDIR',
    syscall: 'read'
  }
} Plugin load failed: %s hexo-theme-landscape

![2020-12-13](https://user-images.githubusercontent.com/18474386/102007299-c2d8a780-3d62-11eb-89a6-f8f32f1d2a4b.png)

## How to reproduce?

* Step1
install a theme by:
`git clone https://github.com/klugjo/hexo-theme-clean-blog.git themes/clean-blog`
* Step2
change the theme to the newly installed one:
```diff
# Extensions
## Plugins: https://hexo.io/plugins/
## Themes: https://hexo.io/themes/
-theme: landscape
+theme: clean-blog
```

then run the commands

## Is the problem still there under "Safe mode"?
Yes

## Environment & Settings

**Node.js & npm version(`node -v && npm -v`)**

```
v14.15.1
6.14.9
```

**Your site `_config.yml`** (Optional)

```
# Hexo Configuration
## Docs: https://hexo.io/docs/configuration.html
## Source: https://github.com/hexojs/hexo/

# Site
title: Hexo
subtitle: ''
description: ''
keywords:
author: John Doe
language: en
timezone: ''

# URL
## If your site is put in a subdirectory, set url as 'http://example.com/child' and root as '/child/'
url: http://example.com
root: /
permalink: :year/:month/:day/:title/
permalink_defaults:
pretty_urls:
  trailing_index: true # Set to false to remove trailing 'index.html' from permalinks
  trailing_html: true # Set to false to remove trailing '.html' from permalinks

# Directory
source_dir: source
public_dir: public
tag_dir: tags
archive_dir: archives
category_dir: categories
code_dir: downloads/code
i18n_dir: :lang
skip_render:

# Writing
new_post_name: :title.md # File name of new posts
default_layout: post
titlecase: false # Transform title into titlecase
external_link:
  enable: true # Open external links in new tab
  field: site # Apply to the whole site
  exclude: ''
filename_case: 0
render_drafts: false
post_asset_folder: false
relative_link: false
future: true
highlight:
  enable: true
  line_number: true
  auto_detect: false
  tab_replace: ''
  wrap: true
  hljs: false
prismjs:
  enable: false
  preprocess: true
  line_number: true
  tab_replace: ''

# Home page setting
# path: Root path for your blogs index page. (default = '')
# per_page: Posts displayed per page. (0 = disable pagination)
# order_by: Posts order. (Order by date descending by default)
index_generator:
  path: ''
  per_page: 10
  order_by: -date

# Category & Tag
default_category: uncategorized
category_map:
tag_map:

# Metadata elements
## https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta
meta_generator: true

# Date / Time format
## Hexo uses Moment.js to parse and display date
## You can customize the date format as defined in
## http://momentjs.com/docs/#/displaying/format/
date_format: YYYY-MM-DD
time_format: HH:mm:ss
## updated_option supports 'mtime', 'date', 'empty'
updated_option: 'mtime'

# Pagination
## Set per_page to 0 to disable pagination
per_page: 10
pagination_dir: page

# Include / Exclude file(s)
## include:/exclude: options only apply to the 'source/' folder
include:
exclude:
ignore:

# Extensions
## Plugins: https://hexo.io/plugins/
## Themes: https://hexo.io/themes/
theme: clean-blog
 

# Deployment
## Docs: https://hexo.io/docs/one-command-deployment
deploy:
  type: git
  repo: https://github.com/755452800/755452800.github.io
  branch: main

```

**Your theme `_config.yml`** (Optional)

```
# Header
menu:
  Home: /
  Archives: /archives
  Tags: /tags
  Categories: /categories
  Github:
    url: https://github.com/klugjo/hexo-theme-clean-blog
    icon: github

# Title on top left of menu. Leave empty to use main blog title
menu_title: Configurable Title

# URL of the Home page image
index_cover: http://www.codeblocq.com/assets/projects/hexo-theme-clean-blog/img/home-bg.jpg

# Default post title
default_post_title: Untitled

# Comments. Choose one by filling up the information
comments:
  # Disqus comments
  disqus_shortname:
  # Facebook comments
  facebook:
    appid:
    comment_count: 5
    comment_colorscheme: light

# Google Analytics Tracking ID
google_analytics:

# Addthis ID
addthis:

# set your own favicon
favicon:

# Social Accounts
twitter_url:
twitter_handle:
facebook_url:
github_url: https://github.com/klugjo/hexo-theme-clean-blog
gitlab_url:
linkedin_url:
mailto:
```

**Hexo and Plugin version(`npm ls --depth 0`)**

```
C:\Users\<myUserName>
`-- (empty)
```

**Your package.json `package.json`**

```
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "scripts": {
    "build": "hexo generate",
    "clean": "hexo clean",
    "deploy": "hexo deploy",
    "server": "hexo server"
  },
  "hexo": {
    "version": "5.2.0"
  },
  "dependencies": {
    "hexo": "^5.0.0",
    "hexo-deployer-git": "^2.1.0",
    "hexo-generator-archive": "^1.0.0",
    "hexo-generator-category": "^1.0.0",
    "hexo-generator-index": "^2.0.0",
    "hexo-generator-tag": "^1.0.0",
    "hexo-renderer-ejs": "^1.0.0",
    "hexo-renderer-marked": "^3.0.0",
    "hexo-renderer-stylus": "^2.0.0",
    "hexo-server": "^2.0.0",
    "hexo-theme-landscape": "^0.0.3"
  }
}
```

**Comment by fgu30** [comment]
I am having the same issue.

**Comment by toastsgithub** [comment]
make sure only one theme package in your `node_modules`, otherwise, hexo will load all other `hexo-theme-*` package as normal plugin, which cause this problem;
Hexo is already solved this problem #4592 , but not released
you guys can workaround this by uninstalling other theme package (ie. hexo-theme-landscape) for right now

**Comment by 755452800** [comment]
> make sure only one theme package in your `node_modules`, otherwise, hexo will load all other `hexo-theme-*` package as normal plugin, which cause this problem;
> Hexo is already solved this problem #4592 , but not released
> you guys can workaround this by uninstalling other theme package (ie. hexo-theme-landscape) for right now

Thanks for answering!
